openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1250,7 +1250,7 @@ This specification also defines a new OAuth 2.0 Authorization Server metadata [@
 
 Credential Issuers often want to know what Wallet they are issuing Credentials to and how private keys are managed for the following reasons:
 
-* The Credential Issuer MAY want to ensure that private keys are properly protected from exfiltration and replay to prevent an adversary from impersonating the legitimate Credential holder by presenting her Credential.
+* The Credential Issuer MAY want to ensure that private keys are properly protected from exfiltration and replay to prevent an adversary from impersonating the legitimate Credential Holders by presenting their Credentials.
```

Comment:
```suggestion
* The Credential Issuer MAY want to ensure that private keys are adequately protected against exfiltration and replay. This precaution helps prevent adversaries from impersonating legitimate Credential Holders and presenting the Credentials.
```

Reply:
the original text is grammatically correct, too :).

Reply:
I'm neutral, I only have tried to resolve the pending comments. If the authors agreed they may reject or merge this last suggestion, no worries.